### PR TITLE
george.petre1104 Project 

### PR DESCRIPTION
**Pull request by GeorgePetre11**
## Pull Request Overview
This pull request adds the documentation for the project RC car.

## TODO or Help Wanted
This pull request still needs feedback.

## Author
Signed-off-by: George-Alexandru Petre [george.petre1104@stud.isb.upb.ro](mailto:george.petre1104@stud.isb.upb.ro)

### CODE REVIEW

File: docs/project/2024/george.petre1104/index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# RC Car controlled with WIFI
+Remote Controlled Car using WIFI and a phone app. 
+
+:::info 
+
+**Author**: George-Alexandru PETRE
```
Comment (alexandruradovici) [inline docs/project/2024/george.petre1104/index.md:6]:
```suggestion
**Author**: George-Alexandru PETRE  \
```

File: docs/project/2024/george.petre1104/index.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/george.petre1104/index.md:89]: Do you need these libraries?

File: docs/project/2024/george.petre1104/index.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://www.youtube.com/watch?v=iTo4Qh2R6m4&list=PLfPN6PiGnry6a4tNFH2uDDkj0rArkB1w-)
+2. [link](https://www.youtube.com/watch?v=U4unGGNjFBg&list=PLfPN6PiGnry6a4tNFH2uDDkj0rArkB1w-&index=4)
```
Comment (alexandruradovici) [inline docs/project/2024/george.petre1104/index.md:96]: Please add some text to the links, otherwise this will display only the word *link*.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please add your email to the pull request template.

**Review by GeorgePetre11 — COMMENTED** [review]
- modified the pico with the correct link and name of the board
- added the links for the library used for the usage of the L289N driver motor

**Review by GeorgePetre11 — COMMENTED** [review]
- removed the unnecessary libraries
- corrected the name for the components in the materials table
- added text to the links

**Comment by GeorgePetre11** [comment]
> Please add your email to the pull request template.

I have updated the pull request.

File: docs/project/2024/george.petre1104/index.md (hunk 1 above)
Comment (alexandruradovici) [reply docs/project/2024/george.petre1104/index.md:6]: This is not resoved, the \ is not in the file.

**Review by GeorgePetre11 — COMMENTED** [review]
removed slash

File: docs/project/2024/george.petre1104/index.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+
+
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [Video 1 from Youtube](https://www.youtube.com/watch?v=iTo4Qh2R6m4&list=PLfPN6PiGnry6a4tNFH2uDDkj0rArkB1w-)
+2. [Video 2 from Youtube](https://www.youtube.com/watch?v=U4unGGNjFBg&list=PLfPN6PiGnry6a4tNFH2uDDkj0rArkB1w-&index=4)
```
Comment (alexandruradovici) [inline docs/project/2024/george.petre1104/index.md:98]:
```suggestion
1. [Raspberry Pi Pico W: WiFi Controlled Robot](https://www.youtube.com/watch?v=iTo4Qh2R6m4&list=PLfPN6PiGnry6a4tNFH2uDDkj0rArkB1w-)
2. [Raspberry Pi Pico - Bluetooth Controlled Car Project](https://www.youtube.com/watch?v=U4unGGNjFBg&list=PLfPN6PiGnry6a4tNFH2uDDkj0rArkB1w-&index=4)
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by GeorgePetre11 — COMMENTED** [review]
removed the empty brackets () , as it created an error when trying to make the pull request

File: docs/project/2024/george.petre1104/index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico WH](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [56 RON](https://ardushop.ro/ro/home/2819-raspberry-pi-pico-wh.html?search_query=pico+wh&results=108) |
+| [Punte H Dubla L298N] | The Driver Motor | [11.53 RON](https://www.sigmanortec.ro/Punte-H-Dubla-L298N-p125423236) |
```
Comment (alexandruradovici) [inline docs/project/2024/george.petre1104/index.md:63]: Links are missing or brackets are extra.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/george.petre1104/index.md (hunk 5 above)
Comment (GeorgePetre11) [reply docs/project/2024/george.petre1104/index.md]: - added the link for L289N
- removed the rest of the square brackets for the components, as they don't have any links

File: docs/project/2024/george.petre1104/index.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
+# RC Car controlled with WIFI
+Remote Controlled Car using WIFI and a phone app. 
+
+:::info 
+
+**Author**: George-Alexandru PETRE
```
Comment (alexandruradovici) [inline docs/project/2024/george.petre1104/index.md:6]:
```suggestion
**Author**: George-Alexandru PETRE  \
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by DanutAldea — APPROVED** [review]

**Pull request merged**